src/extension.ts

```diff
@@ -74,7 +74,7 @@ export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(PartEditorProvider.register(context));
   context.subscriptions.push(PartGraphSelPanel.register(context));
 
-  context.subscriptions.push(CircleViewerProvider.register(context));
+  CircleViewerProvider.register(context);
```

Comment:
From [#1167](https://github.com/Samsung/ONE-vscode/issues/1167#issue-1329445337),

The function is tidy as is, but I am trying to standardize the registration function as below. 😄 

> Point 1. introduce `public static register(context: vscode.ExtensionContext) : void` function
> Point 2. Register each modules with the static function call
> ```
> OneTreeDataProvider.register(context);
> ```